### ISSUE TITLE
Import Errors for Keras Layers in TensorFlow 2.x

### ISSUE THREAD

**Issue by AliAhmadi-Software** [Alireza-Akhavan/deeplearning-tensorflow2-notebooks#2]
**This import is correct in TensorFlow 2.x:**

from keras.models import Model
from keras.layers import Input
from keras.layers import Conv2D, MaxPooling2D
from keras.layers import Activation, Dense, Flatten, Dropout
from keras.layers import BatchNormalization

in this notebook -> [21_cnn-and-price-regression.ipynb](https://github.com/Alireza-Akhavan/deeplearning-tensorflow2-notebooks/blob/master/21_cnn-and-price-regression.ipynb)
